src/main/java/com/shippo/net/APIResource.java

```diff
@@ -53,6 +53,8 @@ private static String className(Class<?> clazz) {
 			return "customs/declaration";
 		} else if (className.equals("carrieraccount")) {
 			return "carrier_account";
+		} else if (className.equals("batch")) {
+			return "batche";
```

Comment:
This is unfortunately true. This function appends `s` at end of the classname to generate the URI. Hence this name :/